src/avro/src/schema.rs

```diff
@@ -1945,11 +1952,6 @@ fn field_ordering_position(field: &str) -> Option<usize> {
 mod tests {
     use super::*;
 
-    #[test]
-    fn test_invalid_schema() {
-        assert!(Schema::parse_str("invalid").is_err());
```

Comment:
did you mean to add this to UNPARSEABLE_SCHEMAS ?

Reply:
oops, yes! good catch -- adding it